openpype/modules/base.py

```diff
@@ -59,6 +59,14 @@
     "example_addons",
     "default_modules",
 )
+# Modules that won't be loaded in AYON mode from "./openpype/modules"
+# - the same modules are ignored in "./server_addon/create_ayon_addons.py"
+IGNORED_FILENAMES_IN_AYON = {
```

Comment:
not sure if this shouldn't be
```suggestion
IGNORED_MODULE_NAMES_IN_AYON = {
```

Reply:
Those are actually filenames.

Reply:
...ok